### PR TITLE
reconnect flag

### PR DESCRIPTION
**Pull request by dr-bonez**
we should consider doing this for backup.target.list as well

### CODE REVIEW

**Comment by dr-bonez** [comment]
marked as a draft until ui component is complete

**Review by DrewAns — APPROVED** [review]

**Comment by DrewAns** [comment]
Front end section is committed and ready for review

**Comment by ProofOfKeags** [comment]
Who needs to take action next on this?

File: setup-wizard/src/app/pages/recover/recover.page.ts [hunk 1]
```diff
@@ -66,7 +66,22 @@ export class RecoverPage {
         })
       })
 
-      const importableDrive = drives.find(d => !!d.guid)
+      if (!this.mappedDrives.length && reconnect.length) {
+        const list = `<ul>${reconnect.map(recon => `<li>${recon}</li>`)}</ul>`
+        const alert = await this.alertCtrl.create({
+          header: 'Warning',
+          message: `One or more devices you connected had to be reconfigured to support the current hardware platform. Please unplug and replug in the following device(s):<br> ${list}`,
```
Comment (dr-bonez) [inline setup-wizard/src/app/pages/recover/recover.page.ts:73]:
```suggestion
          message: `One or more devices you connected had to be reconfigured to support the current hardware platform. Please unplug and replug in the following device(s), then refresh the page:<br> ${list}`,
```

**Pull request merged**